editor/plugins/game_view_plugin.cpp

```diff
@@ -520,41 +538,6 @@ void GameView::set_is_feature_enabled(bool p_enabled) {
 	is_feature_enabled = p_enabled;
 }
 
-void GameView::set_state(const Dictionary &p_state) {
```

Comment:
You could instead replace it with editor layout. It's global for editor.
The project metadata are per-project, not very useful for Game tab.